tests/test_api/test_feeds.py

```diff
@@ -15,16 +15,6 @@ def test_get_feed_present(self, test_client, feed):
 
         data = json.loads(r.data)
         assert data['feed']['title'] == feed.title
-        assert data['feed']['item_count'] == 0
-
-    def test_get_populated_feed_present(self, test_client, feedWithItems):
```

Comment:
Really was just testing the fixture, which we know works. Now that the ```api/feeds/--id--``` call doesn't contain item_count, this test is redundant.